src/data/set/lattice.lean

```diff
@@ -742,10 +742,28 @@ lemma sUnion_eq_univ_iff {c : set (set α)} :
   ⋃₀ c = univ ↔ ∀ a, ∃ b ∈ c, a ∈ b :=
 by simp only [eq_univ_iff_forall, mem_sUnion]
 
+-- classical
+lemma Inter_eq_empty_iff {f : ι → set α} : (⋂ i, f i) = ∅ ↔ ∀ x, ∃ i, x ∉ f i :=
+by simp [set.eq_empty_iff_forall_not_mem]
+
+-- classical
+lemma bInter_eq_empty_iff {f : α → set β} {s : set α} :
+  (⋂ x ∈ s, f x) = ∅ ↔ ∀ y, ∃ x ∈ s, y ∉ f x :=
+by simp [set.eq_empty_iff_forall_not_mem]
+
 -- classical
 lemma sInter_eq_empty_iff {c : set (set α)} :
   ⋂₀ c = ∅ ↔ ∀ a, ∃ b ∈ c, a ∉ b :=
-by simp [set.eq_empty_iff_forall_not_mem, mem_sInter]
+by simp [set.eq_empty_iff_forall_not_mem]
+
+-- classical
+@[simp] theorem Inter_nonempty_iff {f : ι → set α} : (⋂ i, f i).nonempty ↔ ∃ x, ∀ i, x ∈ f i :=
```

Comment:
I'm not sure this should be in the global simp-set. I would rather leave it up to the user to step across of the `nonempty` API "boundary".
```suggestion
theorem Inter_nonempty_iff {f : ι → set α} : (⋂ i, f i).nonempty ↔ ∃ x, ∀ i, x ∈ f i :=
```

Reply:
Looking at the existing theorems, there are (global) simp theorems `nonempty_xUnion`.
Shouldn't we be consistent with it? And also change the names of the theorems to `nonempty_xInter` too?

Reply:
I did change the name, but still not sure what to do with simp